components/layout_2020/replaced.rs

```diff
@@ -431,15 +434,15 @@ impl ReplacedContent {
                     Below(Length),
                     Above(Length),
                 }
-                let violation = |size, min_size, mut max_size: Option<Length>| {
+                let violation = |size: Au, min_size, mut max_size: Option<Length>| {
```

Comment:
Mixing Au and Length here is a bit weird but I guess it will improve when more things become Au